contrib/sbtc/token-key-admin/contracts/sbtc-token-key-admin.clar

```diff
@@ -0,0 +1,115 @@
+
+;; title: sbtc-token-key-admin
+;; version:
+;; summary:
+;; description:
+
+;; traits
+;;
+(impl-trait 'SP3FBR2AGK5H9QBDH3EEN6DF8EK8JY7RX8QJ5SVTE.sip-010-trait-ft-standard.sip-010-trait)
+
+;; token definitions
+;; 
+(define-fungible-token sbtc)
```

Comment:
Suggestion: you can limit the circulating supply of fungible tokens when it makes sense to do so by passing the limit as a second argument to `define-fungible-token`.  Maybe that's appropriate to do here, given that we (1) want to bound the amount of BTC this contract can represent and (2) enforce the 21 million BTC rule (2.1e15 satoshis)?

Reply:
Done.